src/tracing.rs

```diff
@@ -28,6 +28,23 @@ pub fn setup() -> anyhow::Result<()> {
             tracing_subscriber::filter::filter_fn(|metadata| metadata.is_span()),
         ));
 
+        #[cfg(all(feature = "tracing-log-always", feature = "tracing-logger"))]
+        eprintln!(
+            "Both `tracing-log-always` and `tracing-logger` features are enabled at the same time. \
+             This will cause some logs to be printed twice!"
+        );
```

Comment:
We _can_ move this into a `build.rs` to show a warning for it at compile time. Users have no control over this at runtime anyway.

We can probably also improve to only show one log item anyway.

Reply:
Yeah, I've also thought about it. There are two "warning" like this in the `tracing.rs` already. :/

> We can probably also improve to only show one log item anyway.
Not possible/way too complicated in this case. Especially given that these are tracing (i.e., development/debugging) features.

Reply:
I meant [this](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargo-warning). Or even [this](https://doc.rust-lang.org/std/macro.compile_error.html) if we want it more extreme.

Reply:
Yeah, I got it. `compile_error` won't work, though, as it would break `cargo check --all-features`.

Reply:
Oh, I think I remember why I didn't implement a warning in the `build.rs`. IIRC, it's impossible to distinguish `cargo check`/`cargo clippy` from `build.rs`, and so `cargo check --all-features`/`cargo clippy --all-features` would always produce a warning. And considering that it would also require adding `build.rs` to the `qdrant` crate, just to produce an annoying warning, I decided to just get over it and leave a runtime message.

Reply:
Makes a lot of sense. Thanks!